src/importer/PageImporter.js

```diff
@@ -194,7 +194,7 @@ export default class PageImporter {
   }
 
   cleanup(document) {
-    DOMUtils.remove(document, ['script', 'hr']);
+    DOMUtils.remove(document, ['noscript', 'script', 'hr']);
```

Comment:
This is out of the scope for this PR...

Reply:
If provided a html string, than the noscripts are removed, see

https://github.com/adobe/helix-importer/pull/259/files/07a8a5266dbb7116fa88f370f930077a19a44977#diff-b966904efa617d33768f65ed8352810264c7991e6758a9bf9ce41142cb6e8516L300

I thought it makes sense to have the same behaviour if a document object is provided.